temperature.js

```diff
@@ -0,0 +1,14 @@
+// passes js standard test: https://github.com/feross/standard
+const Temperature = {
+  toFahrenheit: (celsius) => {
+    return celsius * (9 / 5) + 32
+  },
+  toCelsius: (fahrenheit) => {
+    return (fahrenheit - 32) * (5 / 9)
+  }
+}
+
+let tempToday = process.env.TEMP || 82
```

Comment:
Using `process.env.TEMP` is a bit weird here. Personally, I think it would read better to use `process.argv[2]`, and call it using `node temperature.js 82`.